src/game/models/vector.rs

```diff
@@ -0,0 +1,144 @@
+use rand;
```

Comment:
Ah, ok. I had thought that rand was provided by rust, but wasn't in std, and was trying to alphabetize it. I'll move it down.